docs/tutorial.md

```diff
@@ -81,40 +80,15 @@ const Page = () => {
 };
 ```
 
-### Add spatial navigation nodes
+### Add spatial navigation focusable view
```

Comment:
ouch maybe the tutorial needs some change, it does not need to be two steps anymore since the focusable view is always... focusable
you should merge these two steps!

and we need to talk about the nodes somewhere else 🤷

Reply:
Done :)